package.json

```diff
@@ -25,14 +29,20 @@
 		"webextension-polyfill": "^0.7.0"
 	},
 	"devDependencies": {
-		"copy-webpack-plugin": "^5.0.3",
+		"@parcel/config-webextension": "^2.0.0-nightly.2220",
+		"@parcel/optimizer-cssnano": "^2.0.0-nightly.599",
+		"@parcel/optimizer-htmlnano": "^2.0.0-nightly.599",
+		"@parcel/packager-css": "^2.0.0-nightly.599",
+		"@parcel/packager-html": "^2.0.0-nightly.599",
+		"@parcel/transformer-css": "^2.0.0-nightly.599",
+		"@parcel/transformer-html": "^2.0.0-nightly.598",
+		"@parcel/transformer-postcss": "^2.0.0-nightly.599",
+		"@parcel/transformer-posthtml": "^2.0.0-nightly.598",
```

Comment:
I don't know why suddenly Parcel is auto-installing these. Previous, identical-ish PRs didn't do this. Parcel works in mysterious ways.

Reply:
Only after https://github.com/parcel-bundler/parcel/pull/5769 😉

Reply:
Makes sense. I haven’t had time to open an issue, but the auto install was really broken. It would nuke node_modules and completely stall when trying to add a second dependency. The loader would stop moving. I had to delete the lock, node_modules and reinstall multiple times until the whole list of dependencies stopped growing.

Reply:
-> https://github.com/parcel-bundler/parcel/issues/5829